src/events/eventToAction.js

```diff
@@ -0,0 +1,119 @@
+import {
+  EVENT_NEW_MESSAGE,
+  EVENT_PRESENCE,
+  EVENT_REACTION_ADD,
+  EVENT_REACTION_REMOVE,
+  EVENT_STREAM_ADD,
+  EVENT_STREAM_REMOVE,
+  EVENT_STREAM_UPDATE,
+  EVENT_SUBSCRIPTION_ADD,
+  EVENT_SUBSCRIPTION_REMOVE,
+  EVENT_SUBSCRIPTION_UPDATE,
+  EVENT_SUBSCRIPTION_PEER_ADD,
+  EVENT_UPDATE_MESSAGE,
+  EVENT_UPDATE_MESSAGE_FLAGS,
+  EVENT_USER_ADD,
+  EVENT_USER_REMOVE,
+  EVENT_USER_UPDATE,
+} from '../constants';
+
+import { getUserById } from '../users/usersSelectors';
+
+const opToActionSubscription = {
```

Comment:
I don't really like this abstraction for a few reasons.
1. It's harder to read and understand because of the indirection
2. It's hard to extend. What if the `add` and `remove` subscription actions have different fields?

I'd prefer we get rid of the `opToAction..` maps and just put the login inline